cmd/root.go

```diff
@@ -57,69 +57,57 @@ func initConfig() {
 	// Check args
 	if len(os.Args) == 1 {
 		return
-	} else if len(os.Args) != 2 {
+	}
+
+	if len(os.Args) == 2 {
```

Comment:
I'm not sure why we put this validation 🤔

Reply:
It is need to run these commands without Datadog credentials.
If remove this, user can't see help message or set new credentials:
```shell
% ./dogleash help
2019/08/02 16:21:25
Dogleash config file does not exist. [/Users/user/.config/dogleash/config.yml]

% ./dogleash config set
2019/08/02 16:21:30
Dogleash config file does not exist. [/Users/user/.config/dogleash/config.yml]
```

Reply:
Ah, I see. Thanks!